src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoPage.java

```diff
@@ -15,4 +17,15 @@
     public float getDefaultZoom() {
         return 1.0f;
     }
+
+    public List<String> informationText() {
+
+        return new ArrayList<String>() {{
```

Comment:
This generates an anonymous subclass of `ArrayList<String>` with statically-initialized members. Perhaps it would make more sense to use something like:
```java
return Stream.of("gregtech.multiblock.preview.tilt", [...])
             .map(I18n::format)
             .collect(Collectors.toList());
```